app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -664,6 +665,40 @@ class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
         }.launch(intent)
     }
 
+    private fun renameCategoryAux(oldCategory: PasswordItem, error: Boolean) {
```

Comment:
Change this to just `renameCategory`? Also add a default value to error here so we can avoid passing false from the public method, the code feels wrong with that in place. A KDoc comment explaining this method and the relevance of the `error` parameter would be helpful from a maintainability aspect.